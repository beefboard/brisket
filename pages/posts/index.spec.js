import VueTestUtils from '@vue/test-utils'
import { shallowMount } from '@vue/test-utils'
import index from './index.vue'

describe('posts/index.vue', () => {
  it('renders properly', () => {})
})
