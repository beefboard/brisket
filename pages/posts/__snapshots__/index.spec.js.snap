// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`posts/index.vue renders properly 1`] = `
<div class="page-container">
  <div class="filters">
    <div>Filter</div>
    <div><input id="new" type="radio" name="filter" value="new"> New
      <div><input id="pinned" type="radio" name="filter" value="pinned"> Pinned
      </div>
      <div><input id="unapproved" type="radio" name="filter" value="unapproved"> Unapproved
      </div>
    </div>
  </div>
  <div class="posts">
    <post-stub post="[object Object]" class="browse-post"></post-stub>
    <post-stub post="[object Object]" class="browse-post"></post-stub>
    <post-stub post="[object Object]" class="browse-post"></post-stub>
  </div>
</div>
`;
